apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/CacheKeyResolver.kt

```diff
@@ -2,6 +2,9 @@ package com.apollographql.apollo.cache.normalized
 
 import com.apollographql.apollo.api.Operation
 import com.apollographql.apollo.api.ResponseField
+import kotlin.jvm.JvmField
+import kotlin.jvm.JvmStatic
+import kotlin.jvm.JvmSuppressWildcards
```

Comment:
unused ?

Reply:
These are used. Somehow in a regular `jvm` project, you can use these without import. But in Multiplatform, they need to be explicitly imported.